From 903ee8264222ee58166c51a8726790acb2f859f6 Mon Sep 17 00:00:00 2001
From: Eric Sauvageau <merlin@asuswrt-merlin.net>
Date: Tue, 23 Jun 2026 11:37:16 -0400
Subject: [PATCH] webui: don't tie traffic analyzer availability to bwdpi

Would be a problem once bwdpi is fully replaced with ark.

(cherry picked from commit 02bae8e038f0d21ca6d862140aecb142c9b9338a)
---
 release/src/router/www/state.js | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/release/src/router/www/state.js b/release/src/router/www/state.js
index b86d9a6b33..efef034f5d 100644
--- a/release/src/router/www/state.js
+++ b/release/src/router/www/state.js
@@ -703,8 +703,7 @@ var ark_qoe_support = isSupport("ark_qoe");
 var adaptiveqos_support = (ark_qoe_support > 0) ? 0 : isSupport("adaptive_qos");
 var ipsec_srv_support = isSupport("ipsec_srv");
 var ipsec_cli_support = isSupport("ipsec_cli");
-//var traffic_analyzer_support = isSupport("traffic_analyzer");
-var traffic_analyzer_support = bwdpi_support;
+var traffic_analyzer_support = isSupport("traffic_analyzer");
 var traffic_limiter_support = isSupport("traffic_limiter");
 var dns_dpi_support = isSupport("dns_dpi");
 var router_boost_support = isSupport("router_boost");
-- 
2.25.1

